src/components/Shared/SendModal/SendLN/SendLN.test.tsx

```diff
@@ -27,20 +28,16 @@ describe("SendLN", () => {
       await screen.findByText("wallet.send_lightning")
     ).toBeInTheDocument();
     expect(await screen.findByLabelText("wallet.invoice")).toBeInTheDocument();
-    expect(
-      await screen.findByRole("button", { name: "wallet.send" })
-    ).not.toBeDisabled();
+    expect(await screen.findByText("wallet.send")).not.toBeDisabled();
   });
 
   test("validates the input for empty value", async () => {
     setup();
     const invoiceInput = await screen.findByLabelText("wallet.invoice");
 
-    expect(
-      await screen.findByRole("button", { name: "wallet.send" })
-    ).not.toBeDisabled();
+    expect(await screen.findByText("wallet.send")).not.toBeDisabled();
 
-    userEvent.click(await screen.findByRole("button", { name: "wallet.send" }));
+    userEvent.click(await screen.findByText("wallet.send"));
```

Comment:
Just a prefrenence or a reason?

Reply:
Tests were failing, so I changed that and it fixed it

Reply:
retried, yes the tests are failing when using findByRole. Most likely because there is now also an icon inside it, not only text and thus the name is not "wallet.send" anymore.